scripts/dotnet-cli-build/Publishing/AzurePublisher.cs

```diff
@@ -187,5 +187,23 @@ public string CalculateArchiveBlob(string archiveFile, string channel, string ve
         {
             return $"{channel}/Binaries/{version}/{Path.GetFileName(archiveFile)}";
         }
+
+        public void DownloadFiles(string blobVirtualDirectory, string fileExtension, string downloadPath)
```

Comment:
Since the parameter is named "fileExtension" calling this DownloadFilesWithExtension seems like it would be redundant.  Also, this is just a utility function that is only used in our build scripts and can easily be renamed/refactored later.

Leaving as is, for now.